2. Web API now calls Microsoft Graph/TodoListService/TodoListService.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Identity.Web" Version="0.2.2-preview" />
+    <PackageReference Include="Microsoft.Identity.Web" Version="0.3.0-preview" />
     <PackageReference Include="Microsoft.Graph" Version="3.8.0" />
```

Comment:
We don't need the Graph SDK any longer.
I even doubt this builds? (package downgrade) ?